CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@ possible.
 - Reach out to us with questions or suggestions on our Slack channel.
 - If you haven't already, complete [the Contributor License Agreement ("CLA")](https://code.facebook.com/cla).
 
-## Contributor License Agreement ("CLA")
+## Contributor License Agreement ("CLA").
```

Comment:
Hi @vineethavg10! Thank you sending the PR.
Regarding the change it looks like we don't use periods for headers in this document.